api/APIREADME.md

```diff
@@ -0,0 +1,112 @@
+# CoTripper API
+
+## Description
+
+CoTripper is a Django-based project aimed at providing an experience-based travel platform for single mom families. CoTripper aims to encourage ideas, exposure, and to be helpful around helping planning vacations and building community. We're more than a travel solution.
+
+## Technologies
+
+- Python
+- Django
+- PostgreSQL
+- Psql
+- Django Rest Framework
+
+## Database Setup
+
+1. Clone the Repository
+
+```
+git clone git@github.com:CotripperPlatform/CoTrip.git
+```
+
+2. Install pipenv and then install psycopg2-binary and autopep8
+
+```
+pipenv install
+
+pipenv install psycopg2-binary
+
+ pipenv install autopep8
+```
```

Comment:
Your thinking here is spot on! This will all be listed in the `Pipfile` - so they should be able to just run `pipenv install` to create a virtual environment and have everything installed